example/cumulus-tf/variables.tf

```diff
@@ -424,7 +424,7 @@ variable "lambda_memory_sizes" {
   type = map(number)
   default = {
     cleanExecutions = 384           # archive
-    DistributionApiEndpoints = 384  # cumulus_distribution
+    DistributionApiEndpoints = 512  # cumulus_distribution
```

Comment:
Might be worth breaking out into a separate PR depending on timing. Fine to leave it here too.

Reply:
Actually, this is a different Lambda than I was thinking. We need to increase for both Discover and Distribution?

Reply:
Yes, distribution is likely due to size bloat in the API package, although I can't prove it.   It didn't run *at all* without this increase.    We can talk more/demo if you'd like.

Reply:
I don't know that we need to 100% prove that. I think it's fine to bump it up.